app/src/main/java/org/sagebionetworks/research/mpower/MainActivity.kt

```diff
@@ -0,0 +1,63 @@
+package org.sagebionetworks.research.mpower
+
+import android.os.Bundle
+import android.support.design.widget.BottomNavigationView
+import android.support.v4.app.Fragment
+import android.support.v7.app.AppCompatActivity
+import android.view.MenuItem
+import dagger.android.AndroidInjection
+import dagger.android.AndroidInjector
+import dagger.android.DispatchingAndroidInjector
+import dagger.android.support.HasSupportFragmentInjector
+import kotlinx.android.synthetic.main.activity_main.navigation
+import org.sagebionetworks.research.mpower.history.HistoryFragment;
+import org.sagebionetworks.research.mpower.insights.InsightsFragment;
+import org.sagebionetworks.research.mpower.profile.ProfileFragment;
+import org.sagebionetworks.research.mpower.logging.LoggingFragment;
+
+import javax.inject.Inject
+
+class MainActivity : AppCompatActivity(), HasSupportFragmentInjector {

```

Comment:
I haven't been keeping up with Android lately, I didn't know they now support Kotlin! I'm really excited to code in it.